test/test-script.sh.in

```diff
@@ -13,10 +13,10 @@ SCRATCH_DIR=/var/tmp/test-casync.$RANDOM
 mkdir -p $SCRATCH_DIR/src
 
 if [ `id -u` == 0 ] ; then
-    cp -a @top_srcdir@ $SCRATCH_DIR/src
+    cp -a @top_srcdir@/test-files $SCRATCH_DIR/src/
```

Comment:
hmm, the intention was actually to copy the whole source tree, so that we actually have some data to test... the "test-files" subdir is usually pretty empty...

I wonder if we can use "cp" for a copy here that just copies the stuff inside of `@top_srcdir@` rather than `@top_srcdir@` altogether